Phases/Assessments/Phase_1.md

```diff
@@ -0,0 +1,37 @@
+# Phase 1 Assessment
+
+The phase 1 assessment will test your ability to write basic JavaScript programs for the Node.js platform as well as your fundamental SQL knowledge.
+
+You will _not_ be assessed on your HTML, CSS, or other web-specific skills.
+
+## Skills Assessed
+
+- Programming (Beginner Level)
+- JavaScript (Beginner Level)
```

Comment:
I don't see how "Programming Fundamentals" adds any more clarity here for folks who lack our familiarity.  I like the "Beginner" distinction because it implies progress through the guild, like the transition from novice to journeyman (journeyperson?).